fractions/microprofile/microprofile-fault-tolerance/pom.xml

```diff
@@ -35,6 +35,7 @@
     <swarm.fraction.tags>Eclipse MicroProfile,MicroServices,Fault Tolerance</swarm.fraction.tags>
     <version.arquillian-weld-embedded>2.0.0.Beta5</version.arquillian-weld-embedded>
     <version.awaitility>3.1.0</version.awaitility>
+    <version.smallrye-fault-tolerance>1.0.0</version.smallrye-fault-tolerance>
```

Comment:
This should go into the `build-parent` pom:
https://github.com/thorntail/thorntail/blob/master/build-parent/pom.xml#L112-L116

Reply:
Also add `io.smallrye:smallrye-fault-tolerance` dependency to the `<dependencyManagement>` section in build-parent; there is a `<!-- SmallRye -->` comment where other MPs reside.